### ISSUE TITLE
Problem with sending URLs through CLI

### ISSUE THREAD

**Issue by vodkawithcoke** [andyholmes/gnome-shell-extension-gsconnect#995]
When running the script (with all variables assigned):

```${GSCONNECT} -d ${DEVICE_ID} --share-file="/path/to/file"```

It works perfectly and transfers the named file to my mobile device, without any problem. But when the last line of the script is changed to:

```${GSCONNECT} -d ${DEVICE_ID} --share-link="www.google.com"```

it returns the error:

```
(gsconnect:76593): Gjs-WARNING **: 02:23:37.868: JS ERROR: Error: Error invoking toString, at argument 0 (byteArray): Not an object
_cliShareLink@/home/USER/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/daemon.js:648:37
vfunc_handle_local_options@/home/USER/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/daemon.js:716:22
@/home/USER/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/daemon.js:729:17
```

Point to be noted: sending link works flawlessly when used from browser extension (for both Chrome and Firefox).
I looked everywhere but I couldn't find any solution to this specific CLI problem. It there something I might be doing wrong? I'm new to javascript, and bash.

I'm running:
- GSConnect version **43** (installed from Gnome Extensions website)
- Ubuntu **20.04.1** LTS
- GNOME Version **3.36.3**
- KDE Connect Version **1.14.2**

**Comment by andyholmes** [comment]
Thanks for reporting, should be fixed in b868e5c. Zip will be available on the [CI job](https://github.com/andyholmes/gnome-shell-extension-gsconnect/actions/runs/292330788).

**Comment by vodkawithcoke** [comment]
It works now. Thankyou!
